modules/lvms-about-lvmcluster-cr.adoc

```diff
@@ -0,0 +1,110 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="about-lvmcluster_{context}"]
+= About the LVMCluster custom resource
+
+After you have installed {lvms}, you must create an `LVMCluster` custom resource (CR) on a worked node.
+
+You can configure the `LVMCluster` CR to perform the following actions:
+
+* Create LVM volume groups.
```

Comment:
Updated it as follows:
"Create LVM volume groups that you can use to provision persistent volume claims (PVCs).
Excluded the "CSIDriver" as it might be information overload in the context of this module.